source-code/main.py

```diff
@@ -55,7 +56,11 @@ def get_ssm_parameter_value(self, name: str) -> str:
             Name=name,
             WithDecryption=True
         )
-        return response.get('Parameter').get('Value')
+        returnval = response.get('Parameter').get('Value')
+        if (name.startswith('/aws/reference/secretsmanager')):
+            val = json.loads(returnval)
```

Comment:
Should we handle the case when secret stores just password instead of a json?
```python
try:
  val = json.loads(returnval)
  returnval = val['password']
except ValueError as e:
  pass
```

Reply:
That makes sense.